core/src/main/resources/jenkins/model/JobGlobalBuildDiscarderStrategy/config.properties

```diff
@@ -0,0 +1,3 @@
+blurb = Build discarders configured for a job are only run after a build finishes. \
+  This option runs jobs'' configured build discarders periodically, applying configuration changes even when no new builds are run. \
```

Comment:
> applying configuration changes even when no new builds are run.

This wording seems a bit confusing to me. From where will configuration changes that can be applied to the job come, if not from the job itself?

What kind of configuration change could exist but in an "unapplied" state, that this option would apply?

Should it read 

> applying build discarder rules even when no new builds are run

or is it really talking about job configuration?

Reply:
> or is it really talking about job configuration?

Well, build discarder rules are a part of the job configuration. So in a sense, yes.

Currently, you'd need to run a new build to apply a changed build discarder configuration, since it's only consulted after a build finishes. So without new builds, you can change this option and it doesn't have any consequences. For an obsolete/archived job, it would never actually delete builds/artifacts.

Perhaps

> …applying changes to a job's build discarder configuration even when…

?